queries.sql

```diff
@@ -137,4 +137,63 @@ FROM owners
     JOIN animals ON animals.owners_id = owners.id
 GROUP BY owners.id
 ORDER BY num_of_animals DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
```

Comment:
![image](https://github.com/misalazeem/Vet-Clinic-Database/assets/93039359/d4fb89f6-117a-415b-b991-52a9cdd91a06)

- [ ] It would be nice if you fix your LinkedIn link because right now it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍